test/asset_manager_test.rb

```diff
@@ -54,6 +54,14 @@
     end
   end
 
+  it "raises not found when a whitehall asset does not exist" do
```

Comment:
Minor: Again the capitalisation of "whitehall" in the two new test names is inconsistent with other tests.